containers/node-burn/Containerfile

```diff
@@ -20,11 +20,17 @@ RUN git clone https://github.com/eth-cscs/node-burn.git && \
         sed -i 's/BLAS::BLAS/nvpl::lapack_lp64_omp/' CMakeLists.txt; \
     fi && \
     cmake -DCMAKE_BUILD_TYPE=Release -DTF32=ON -DDOUBLE=OFF -B build && \
-    cmake -DCMAKE_BUILD_TYPE=Release -B build-f64&& \
+    cmake -DCMAKE_BUILD_TYPE=Release -B build-f64 && \
```

Comment:
1/ line 10: apt install nvpl
but line 44: apt install libgomp1 nvpl-blas
- what's the difference between nvpl and nvpl-blas, should it be the same ?
2/ you build fp32 but never test it